doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -154,6 +154,10 @@ Build the ACRN Hypervisor on Ubuntu
         bison
 
       $ sudo pip3 install kconfiglib
+      $ wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
```

Comment:
This is already explained in this document (further down). If you want to make sure people don't miss this step, add a pointer to the paragraph but better not to repeat it (and worse, repeat it using different instructions :-()

Reply:
> This is already explained in this document (further down). If you want to make sure people don't miss this step, add a pointer to the paragraph but better not to repeat it (and worse, repeat it using different instructions :-()

ok, I add a pointer to the paragraph, and updated it.